src/index.js

```diff
@@ -1,6 +1,16 @@
 const { clipboard } = require('electron')
 const sanitizeInput = require('./sanitizeShellInput')
 
+const isPasting = e => {
+  const isDarwin = process.platform === "darwin";
+  const key = e.which || e.keyCode; // keyCode detection
```

Comment:
do we need e.which? according to [this](http://stackoverflow.com/questions/4471582/javascript-keycode-vs-which) it's only for *some* browsers. In electron i don't think that we need to care about it.